client/components/token-field/suggestions-list.jsx

```diff
@@ -1,8 +1,9 @@
 /**
  * External dependencies
  */
-var map = require( 'lodash/map' ),
-	React = require( 'react' ),
+import { map } from 'lodash';
+
```

Comment:
It'd be nice to not have this extra space, but I suppose https://github.com/benjamn/recast/issues/405 isn't fixed yet